docs/rst/manual/getting_started/configuration/poly_new/guide.rst

```diff
@@ -0,0 +1,459 @@
+*******************************
+Now you should delete the ``storage`` and ``poly_out`` folders from last run,
+so we know we are getting only new data.
+
+Launch ERT again. Notice that the config summary now specifies the name of the
+parameter set we defined. Then select Ensemble Experiment in the simulation
+mode, and start the simulation.
+
+When the simulations are done, you can now press the "Create Plot" button in
+the progress window or in the main window, and the Plotting window should open.
+Here you can now see the distributions of the three different parameters we
+created. They are named ``COEFFS:COEFF_A``, ``COEFFS:COEFF_B`` and
+``COEFFS:COEFF_C``, with the parameter set name first, then a colon and then
+the name of the specific parameter.
+
+You should see something similar to this:
+
+    .. image:: with_parameters/plots.png
+
+Play around and look at the different plots.
+
+Inspecting the parameters and results
+*************************************
+
+Inside each of the runpaths you should now be able to find the instantiated
+parameter template files ``coeffs.json``. Looking at them (e.g. with
+``cat poly_out/real_4/iter_0/coeffs.json``) you should see something like
+this::
+
+    {
+        "a": 0.830303,
+        "b": 1.69181,
+        "c": 0.114524
+    }
+
+If you now look at the generated file ``poly_0.out`` in the runpaths you should
+also see that each simulation has yielded different results. Here is one
+possible output from running ``cat poly_out/real_0/iter_0/poly_0.out``::
+
+    2.23622
+    4.288035
+    6.83408
+    9.874355
+    13.40886
+    17.437595
+    21.96056
+    26.977755
+    32.48918
+    38.494835
+
+In the next section, we will see how to describe the results to ERT, and how to
+specify some observations that we wish ERT to optimise towards.
+
+Reading simulation results
+--------------------------
+We have to tell ERT where to find the results of our simulations. For general
+data like we have in this example, we use the :ref:`GEN_DATA <gen_data>`
+keyword. Add this line to the ``poly.ert`` file::
+
+    GEN_DATA POLY_RES RESULT_FILE:poly_%d.out REPORT_STEPS:0 INPUT_FORMAT:ASCII
+
+The arguments of ``GEN_DATA``:
+
+POLY_RES
+    Name of this result set.
+RESULT_FILE:poly_%d.out
+    File with results of simulation. The ``%d`` is always ``0``, but needs to
+    be specified. (it was used in earlier ERT versions)
+REPORT_STEPS:0
+    TODO
+INPUT_FORMAT:ASCII
+    Specifies that the file is a normal text file (`ASCII` stands for "American
+    Standard Code for Information Interchange")
+
+The config file should now look like this:
+
+    .. literalinclude :: with_results/poly.ert
+
+If you now run the ensemble experiment again, and then open the plot view, you
+should see a new plot available called ``POLY_RES``:
+
+    .. image :: with_results/poly_plot.png
+
+Adding observations
+-------------------
+In order to use the update algorithms of ERT, we need to have some observations
```

Comment:
`To use the update algorithms of ERT, we need to have some observations to compare with the results.` More to the point?